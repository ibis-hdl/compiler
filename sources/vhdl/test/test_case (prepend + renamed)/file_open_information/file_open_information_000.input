-- VHDL test case input
OPEN file_open_kind IS "file_logical_name"

