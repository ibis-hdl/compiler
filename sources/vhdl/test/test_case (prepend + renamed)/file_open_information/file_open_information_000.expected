(file_open_information
  (expression
    (relation
      (shift_expression
        (simple_expression
          (term
            (primary<v>
              (name<v>
                (identifier
                  file_open_kind
                )
              )
            )
          )
        )
      )
    )
  )
  (expression
    (relation
      (shift_expression
        (simple_expression
          (term
            (primary<v>
              (literal<v>
                (string_literal
                  file_logical_name
                )
              )
            )
          )
        )
      )
    )
  )
)
